test/controllers/geocoder_controller_test.rb

```diff
@@ -223,6 +223,18 @@ def test_identify_latlon_sw_dms
     end
   end
 
+  ##
+  # Test identification of lat/lon pairs with values close to zero
+  def test_identify_latlon_close_to_zero
+    [
+      "0.0000123 -0.0000456",
+      "+0.0000123 -0.0000456",
+      "N 0° 0' 0.4428\", W 0° 0' 1.6416\""
+    ].each do |code|
+      latlon_check code, 0.0000123, -0.0000456
+    end
+  end
+
```

Comment:
This one is going to fail with `TypeError: can't convert nil into BigDecimal`

```suggestion

  ##
  # Test identification of integer lat/lon pairs using N/E with degrees
  def test_identify_latlon_ne_d_int_deg
    [
      "N50° E14°"
    ].each do |code|
      latlon_check code, 50, 14
    end
  end
```

[Used to work correctly](https://www.openstreetmap.org/search?query=N50%C2%B0%20E14%C2%B0)

Reply:
Ha that's effectively an old lurking bug - the first two regexps that are supposed to match the degrees only case but didn't allow the optional degree symbol while the rest of the regexps wrongly made everything after degrees optional so would over match.

All fixed now hopefuly...